Data/Sys/GameSettings/GED.ini

```diff
@@ -2,6 +2,7 @@
 
 [Core]
 # Values set here will override the main Dolphin settings.
+CPUThread = 0
```

Comment:
I know you guys don't like dual-core since it's a hack, but AFAICT setting deterministic dual core to "fake-completion" also fixes the hang at startup in Eternal Darkness and it's more, err, user-friendly for people running Dolphin with weak hardware...